ports/triplets/x64-osx-asan.cmake

```diff
@@ -8,8 +8,11 @@ set(VCPKG_OSX_ARCHITECTURES x86_64)
 set(VCPKG_C_FLAGS "-fsanitize=address")
 set(VCPKG_CXX_FLAGS "-fsanitize=address")
 
-# Hide symbols in the AWS SDK. Fixes symbol collisions with other libraries
-# like arrow (https://github.com/apache/arrow/issues/42154).
-if("${PORT}" MATCHES "^aws-")
-    set(VCPKG_CMAKE_CONFIGURE_OPTIONS "-DCMAKE_CXX_VISIBILITY_PRESET=hidden;-DCMAKE_C_VISIBILITY_PRESET=hidden")
+# Fix CMake 4.0 errors in ports supporting very old CMake verions.
+set(VCPKG_CMAKE_CONFIGURE_OPTIONS "-DCMAKE_POLICY_VERSION_MINIMUM=3.5")
+
+# Hide symbols in spdlog. Fixes symbol collisions in downstream projects
+# (https://github.com/gabime/spdlog/pull/3322 was rejected).
+if("${PORT}" MATCHES "spdlog")
+    list(APPEND VCPKG_CMAKE_CONFIGURE_OPTIONS "-DCMAKE_CXX_VISIBILITY_PRESET=hidden;-DCMAKE_C_VISIBILITY_PRESET=hidden")
 endif()
```

Comment:
This is no longer necesssary [since aws-c-common 0.10.0](https://github.com/awslabs/aws-c-common/compare/v0.9.31...v0.10.0#diff-9b6ea2bf68ed273a9e82a3ef8e0b10a0b3105c67f71719cc879d27c0bcce8dafR255-R261). This flag flows to all other AWS libraries.